spring-cloud-app-broker-core/build.gradle

```diff
@@ -29,13 +29,14 @@ dependencyManagement {
 }
 
 dependencies {
-	implementation project(":spring-cloud-app-broker-deployer")
+	api project(":spring-cloud-app-broker-deployer")
+	api("io.projectreactor:reactor-core")
+	api("org.apache.commons:commons-text:1.4")
+	api("org.springframework.cloud:spring-cloud-open-service-broker-core:${openServiceBrokerVersion}")
```

Comment:
We should probably be using `spring-cloud-starter-open-service-broker` here. Also, this dependency is duplicated below as an `implementation` dependency.

Reply:
I like the idea of using `spring-cloud-open-service-broker-core` as the minimum (transitive) dependency in `spring-cloud-app-broker-core` and having the app broker starter depend on the `spring-cloud-starter-open-service-broker` (as things are now). This makes extensions like `spring-cloud-app-broker-security-credhub` cleaner, in that they can depend on `-core` and only get necessary dependencies without also getting auto-config etc.